CONTRIBUTING.md

```diff
@@ -0,0 +1,114 @@
+# Contributing
+
+Thanks so much for thinking of contributing to `neo4j-graphql-js`, we really
+appreciate it! :heart:
+
+## Get in Touch
+
+There's an active [mailing list](https://groups.google.com/forum/#!forum/neo4j)
+and [Slack channel](https://neo4j.com/slack) where we work directly with the
+community.
+If you're not already a member, sign up!
+
+We love our community and wouldn't be where we are without you.
+
+## Getting Set Up
+
+Clone the repository, install dependencies and build the project:
+
+```
+git clone git@github.com:neo4j-graphql/neo4j-graphql-js.git
+cd neo4j-graphql-js
+npm install
+npm run build
+```
+
+### Testing
+
+We use the `ava` test runner. Run the tests with:
+
+```
+npm run test
+```
+
+The `npm test` script will run unit tests that check GraphQL -> Cypher
+translation and the schema augmentation features and can be easily run locally
+without any extra dependencies.
+
+Full integration tests can be found in `/test` and are
+[run on CircleCI](https://circleci.com/gh/neo4j-graphql/neo4j-graphql-js) as
+part of the CI process.
+
+#### Integration Testing
+
+If you want to run integration tests locally, make sure your setup meets the
+following requirements:
+
+- A local Neo4J instance with username `neo4j` and password `letmein`
+- Your Neo4J instance runs on [this database](https://s3.amazonaws.com/neo4j-sandbox-usecase-datastores/v3_5/recommendations.db.zip)
```

Comment:
@johnymontana thank you for providing the file. Could you upload a dump of your database to `s3.amazonaws.com..`?

```
neo4j-admin dump --to=./recommendations-db
# upload recommendations-db to amazon aws
```
We could improve the documentation here and use `neo4j-admin load ...`.

Reply:
By now I would prefer to check the file into version control. Opinions?

Reply:
I'd prefer not to check the datastore into git. Let's plan for a long term goal of refactoring the tests to use Cypher script fixtures rather than loading the datastore.

In the meantime a `.dump` version of the datastore is available here: https://s3-us-west-2.amazonaws.com/neo4j-datasets-public/recommendations.dump